### PR TITLE
New - Palo Alto Sensor Documentation

### PR DESCRIPTION
**Pull request by skalavala**
I did a [PR](https://github.com/home-assistant/home-assistant/pull/12910) on the main branch for the `Palo Alto` sensor, this is the supporting documentation.

**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12910

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by skalavala** [comment]
If someone can tell me where to upload the logo `palo_alto.png`, that would be great. I've added the logo in the documentation page, but not sure where that goes.

**Comment by frenck** [comment]
@skalavala `source/images/supported_brands/`

File: source/_components/sensor.paloalto.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+To enable this sensor, add the following lines to your `configuration.yaml` file for a GET request:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: paloalto
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.paloalto.markdown:27]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/sensor.paloalto.markdown [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+The `paloalto` sensor platform is displaying the statistical summary of a [Palo Alto](https://www.paloaltonetworks.com/) device.
+
+To enable this sensor, add the following lines to your `configuration.yaml` file for a GET request:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: paloalto
```
Comment (frenck) [inline source/_components/sensor.paloalto.markdown:24]: Seems like the required configuration variables are missing in this example? (`ip_address`, `api_key`)

**Review by frenck — CHANGES_REQUESTED** [review]
@skalavala Left some comments, could you please take a look? 👍

**Comment by frenck** [comment]
Rebased the PR in order to fix the build issue.

@skalavala Friendly reminder, could you please take a look at the remaining requests?

**Comment by skalavala** [comment]
I have closed my PR for the new Palo Alto sensor for now, and when I get to that, I will re-open this PR.

**Pull request closed without merging**